docs/content/preview/contribute/docs/all-page-elements.md

```diff
@@ -0,0 +1,414 @@
+
+Everything that follows is in this tab's file. If you change tabs, you get a whole new page.
+
+### In-page tab widget
+
+This tab widget doesn't use separate files to fill in the content and then link between. Here the content is placed inside the `tabpane` shortcode.
+
+{{< tabpane text=true >}}
+
+  {{% tab header="Java" lang="java" %}}
+
+Another tab style; here all the tab contents are in-page.
+
+### Headings inside this widget don't show up in RightNav
+
+To build and run the application, do the following:
+
+1. Clone the sample application to your computer:
+
+    ```sh
+    git clone https://github.com/YugabyteDB-Samples/yugabyte-simple-java-app.git && cd yugabyte-simple-java-app
+    ```
+
+1. Provide connection parameters.
+
+    The application needs to establish a connection to the YugabyteDB cluster. To do this:
+
+    - Set the following configuration parameters:
+
+        - **host** - the host name of your YugabyteDB cluster. To obtain a YugabyteDB Managed cluster host name, sign in to YugabyteDB Managed, select your cluster on the **Clusters** page, and click **Settings**. The host is displayed under **Connection Parameters**.
+        - **port** - the port number that will be used by the JDBC driver (the default YugabyteDB YSQL port is 5433).
+
+    - Save the file.
+
+1. Start the application.
+
+    ```sh
+    $ java -cp target/yugabyte-simple-java-app-1.0-SNAPSHOT.jar SampleApp
+    ```
+
+The end of this tab.
+
+  {{% /tab %}}
+
+  {{% tab header="Go" lang="go" %}}
+
+The contents of the next tab. You can keep adding tabs in similar fashion.
+
+### Headings inside this widget don't show up in RightNav
+
+To build and run the application, do the following:
+
+1. Clone the sample application to your computer:
+
+    ```sh
+    git clone https://github.com/YugabyteDB-Samples/yugabyte-simple-go-app.git && cd yugabyte-simple-go-app
+    ```
+
+1. Provide connection parameters.
+
+  {{% /tab %}}
+
+{{< /tabpane >}}
+
+### Pills
+
+These pills automatically re-flow based on page width; just keep adding list items. Use these pills for landing pages or as an alternative to tabs.
+
+<ul class="nav yb-pills">
+
+  <li>
+    <a href="../../../develop/build-apps/java/cloud-ysql-yb-jdbc/" class="orange">
+      <i class="fa-brands fa-java"></i>
+      Java
+    </a>
+  </li>
+
+  <li>
+    <a href="../../../develop/build-apps/go/cloud-ysql-go/" class="orange">
+      <i class="fa-brands fa-golang"></i>
+      Go
+    </a>
+  </li>
+
+  <li>
+    <a href="../../../develop/build-apps/python/cloud-ysql-python/" class="orange">
+      <i class="fa-brands fa-python"></i>
+      Python
+    </a>
+  </li>
+
+  <li>
+    <a href="../../../develop/build-apps/nodejs/cloud-ysql-node/" class="orange">
+      <i class="fa-brands fa-node-js"></i>
+      NodeJS
+    </a>
+  </li>
+
+</ul>
+
+## Table
+
+The following is a basic markdown table.
+
+| Table | A column |
+| :--- | :--- |
```

Comment:
Making the alignment markers line up with the column headings makes things easier.
```suggestion
| Table | A column |
| :---- | :------- |
```